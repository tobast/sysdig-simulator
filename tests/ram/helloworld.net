INPUT
OUTPUT once, z
VAR
	a:8, z:8, data:8, once, regout, fbit
IN
a = 0b00000000
data = 0b01010100
fbit = 1
regout = REG fbit
once = NOT regout
z = RAM 8 8 a once a data
