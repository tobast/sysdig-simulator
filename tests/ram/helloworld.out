100000000
001010100
001010100
